__tests__/posts.ts

```diff
@@ -734,8 +734,8 @@ describe('mutation banPost', () => {
 
 describe('mutation reportPost', () => {
```

Comment:
Can you add a test that doesn't send a comment so we'll check for backward compatibility?